build/NuGetPackages/CreateNuGetPackages.proj

```diff
@@ -85,5 +85,9 @@
       <!-- Override the NuGet package version provided by Nerdbank.GitVersioning -->
       <NuGetPackageVersion>$(MajorMinorVersion).$(BuildNumber)$(PrereleaseVersion)</NuGetPackageVersion>
     </PropertyGroup>
+
+    <PropertyGroup Condition=" '$(PublicRelease)' == 'true' ">
+      <NuGetPackageVersion>$(MajorMinorVersion).$(BuildNumber)</NuGetPackageVersion>
```

Comment:
This looks like it will strip out the prerelease tag from your version for public releases, even if it was supposed to be a prerelease. With this as it is, how will you create a 15.1-beta package when -beta keeps getting stripped off of your public releases?